tests/test_auxtel_latiss_take_sequence.py

```diff
@@ -0,0 +1,176 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+import contextlib
+import types
+import unittest
+import logging
+
+import pytest
+from lsst.ts import salobj
+from lsst.ts.standardscripts.auxtel import LatissTakeSequence
+from lsst.ts.standardscripts import BaseScriptTestCase, get_scripts_dir
+
+
+logger = logging.getLogger(__name__)
```

Comment:
Instead of doing this, add the logger to the class.. 

```
    @classmethod
    def setUpClass(cls) -> None:
        cls.log = logging.getLogger(type(self).__name__)
```

Then replace all calls to `logger` to `self.log`...